examples/example-phase1/src/app/(pages)/settings/blog/page.tsx

```diff
@@ -0,0 +1,27 @@
+// Services
+import { getBlogItems } from "@/services/blog";
+
+// Types
+import { BlogItemType } from "@/types";
+
+// Components
+import BlogItem from "./blogItem/page";
+
+const Blog = async () => {
+  const blogs: BlogItemType[] = await getBlogItems()
+
+  return (
+    <div className="flex flex-col items-center">
+      <h1 className="text-xl my-4">Blog page</h1>
+      <ul className="flex flex-row justify-between w-80">
```

Comment:
Separate this list into a component and call `getBlogItems()` inside that. Then apply **Streaming** in the page.